build/yaml/botbuilder-js-daily.yml

```diff
@@ -8,25 +8,13 @@ pool:
 variables:
   NodeVersion: 12.x
   YARN_CACHE_FOLDER: $(Pipeline.Workspace)/.yarn
-#  PackageVersion: Define this in Azure to be settable at queue time. Ex: 4.11.0-dev.{DateStamp}.{CommitHash} 
 
 steps:
-- powershell: |
-   # Replace {DateStamp} and {CommitHash} tokens with the actual values in vars PackageVersion
-   $dateStamp = (Get-Date -format "yyyyMMdd");
-   $commitHash = "$(Build.SourceVersion)".SubString(0,7);
-
-   "Raw PackageVersion = $(PackageVersion)";
-   $v = "$(PackageVersion)".Replace("{DateStamp}",$dateStamp).Replace("{CommitHash}",$commitHash);
-   Write-Host "##vso[task.setvariable variable=PackageVersion;]$v";
-   "Resolved PackageVersion = $v";
-  displayName: resolve variables
-
 - template: templates/build.yml
 
 - template: templates/test.yml
 
-- script: yarn update-versions $(PackageVersion)
+- script: yarn update-versions "$(PackageVersion)" --preview "$(PreviewText)" --deprecated "$(DeprecatedText)" --date "$(DateFormat)" --git "$(IncludeGitSha)"
```

Comment:
While I appreciate the configurability, is the deprecated text going to be dynamic over time?

Reply:
It's possible in certain cases that we'll want to omit it, so supporting a default of `"deprecated"` in the Pipeline but allowing it to be overridden with `""` to omit was my intention.